eng/scripts/Compare-CurrentToCodegeneration.ps1

```diff
@@ -181,7 +181,8 @@ $generateScript = {
       }
 
       # Update code snippets before comparing the diff
-      $mvnOutput = (& mvn --no-transfer-progress codesnippet:update-codesnippet 2>&1)
+      # TODO: remove spotless:apply when issue fixed in emitter
+      $mvnOutput = (& mvn --no-transfer-progress codesnippet:update-codesnippet spotless:apply 2>&1)
```

Comment:
codegen via script/pipeline should be fine, as it does `package` which include `spotless:apply`

for user run `tsp-client update` locally, they would need to format via `spotless:apply` (or call `mvn package` somehow)